app-android-xamarin/Moodis/Moodis/Feature/Group/GroupListAdapter.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+using Android.App;
+using Android.Content;
+using Android.OS;
+using Android.Runtime;
+using Android.Support.V7.Widget;
+using Android.Views;
+using Android.Widget;
+
+namespace Moodis.Feature.Group
+{  
+    class GroupListAdapter : RecyclerView.Adapter
+    {
+        readonly List<Group> groups;
+        readonly Context context;
+        TextView noGroupLabel;
+
+        public GroupListAdapter(Context context,List<Group> groups, TextView noGroupLabel)
+        {
+            this.groups = groups;
+            this.context = context;
+            this.noGroupLabel = noGroupLabel;
+        }
+
+        public override int ItemCount
+        {
+            get { return groups.Count(); }
+        }
+
+        public override void OnBindViewHolder(RecyclerView.ViewHolder holder, int position)
+        {
+            var viewHolder = holder as GroupViewHolder;
+            viewHolder.GroupNameLabel.Text = groups[position].Groupname;
+            viewHolder.GroupNameLabel.Click += (sender, e) =>
+            {
+                var MyIntent = new Intent(context, typeof(GroupMembersActivity));
+                MyIntent.PutExtra("clicked", position);
```

Comment:
is github showing nonsense or this wasn't fixed?

Reply:
I remember fixing it 💩